sdk.go

```diff
@@ -18,9 +21,9 @@ type SDK struct {
 }
 
 // New creates a default SDK object
-func New(address string) *SDK {
+func New(c *rpc.Client) *SDK {
```

Comment:
In general is a good practice in go to accept an interface and return a struct. Here we should receive an RPCClient client instead, like:
```
func New(c RPCClient) *SDK {
```